doc/source/examples/networktheory/CPW media example.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CPW media example\n",
+    "\n",
+    "Coplanar waveguide (CPW) is a planar transmission line were the return conductor is a plane located on both side of a strip conductor on the same layer. The characteristic impedance depends of the strip width and the gap with the return plane, giving two degree of freedom to designer.\n",
+    "\n",
+    "A great advantage of CPW is the convenience to perform shunt or serie connexions with components, because both the signal and the return path are on the same side, thus no via are required. On the other hand, the larges return planes consume a lot of surface on the circuit.\n",
+    "\n",
+    "Coplanar waveguide has quasi-TEM mode and less dispertion than microstrip. However, to avoid effect of slotline-mode, the planes on both side of the strip have to be connected together trough bond-wires. Another solution is to add a conductor plane on the backside of substrate and connect all planes together with via fences. But should the height of the sustrate be small and the gap with coplanar planes be big, then the line would behave like a microstrip line instead.\n",
```

Comment:
typo: "dispersion". "to avoid effect of" -> "to avoid propagation of parasitic slotline mode".

Many MMICs have air bridges that are used to connect ground planes together. On PCB vias can be also used. I would just say that the planes need to be connected together to avoid propagation of parasitic slotline mode.

typo: "substrate".

Reply:
rephrased.